packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.base.tsx

```diff
@@ -104,8 +105,9 @@ export class CheckboxBase extends React.Component<ICheckboxProps, ICheckboxState
     return (
       <KeytipData keytipProps={keytipProps} disabled={disabled}>
         {(keytipAttributes: any): JSX.Element => (
-          <div className={this._classNames.root} title={title}>
+          <div role="presentation" className={this._classNames.root} title={title}>
             <input
+              role={role}
```

Comment:
Isn't this already included as part of `inputProps`

Reply:
good question.

Reply:
If inputProps either has it explicitly or is utilizing nativeProps then it should have it

Reply:
fixed